implementation/gate/src/main/java/eu/efti/eftigate/controller/IdentifiersController.java

```diff
@@ -24,7 +24,7 @@ public class IdentifiersController implements IdentifiersControllerApi {
 
     @Override
     public ResponseEntity<RequestUuidDto> getIdentifiers(final @RequestBody SearchWithIdentifiersRequestDto identifiersRequestDto) {
-        log.info("POST on /getIdenifiers with param vehicleID {}", identifiersRequestDto.getVehicleID());
+        log.info("POST on /getIdentifiers with param vehicleID {}", identifiersRequestDto.getIdentifier());
```

Comment:
Here it says "with param vehicleID" which doesn't match what we are requesting. We could print all of the request parameters?